FableForest merge shop [MEM]

Hassan0Kamel opened the pull request:
another merge shop that no one will use lmao

Hassan0Kamel committed:
Add files via upload

another merge shop that no one will use lmao

wtffidy commented on Other/MergeShops/FableForestMergeShop.cs:
```diff
@@ -0,0 +1,229 @@
+                    Core.FarmingLogger(req.Name, quant);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.BuyItem("fableforest", 814, "Plain Dragon Tail", 1);
```
quantities of 1 are already pre-filled as 1, so you don't need to include that part.

wtffidy commented on Other/MergeShops/FableForestMergeShop.cs:
```diff
@@ -0,0 +1,229 @@
+                    Core.FarmingLogger(req.Name, quant);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.BuyItem("fableforest", 814, "OakHeart ArmBlades", 1);
```
you can just do the buyline(use Adv.BuyItem if the item has *any* gold requirement, you also dont need the while if the buy item is the same as the case.

wtffidy replied:
actualy im seeing muliple buy bits so uu can combine cases *just stack them like:

case "this":
case "that":
case "another one"
case "I'm not typing anymore of these"

Hassan0Kamel committed:
Update FableForestMergeShop.cs

made the cases of buy items stacked + used Adv.BuyItem instead of Core.BuyItem

Hassan0Kamel committed:
Update FableForestMergeShop.cs

done editing the script to make the stacked Cases only use one line Adv.BuyItem

Hassan0Kamel committed:
Update FableForestMergeShop.cs

done changing Req.Name > req.Name 
and deleted the top thing that got generated by itself

wtffidy reviewed (approved):
🥔 🤗
